Containerfile.aws-load-balancer-operator-bundle

```diff
@@ -38,6 +38,26 @@ LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.16.0+git
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 
+# should this operator be supported on OCP 4.4 and earlier (old appregistry format)
+LABEL com.redhat.delivery.backport: false
+
+# this is a bundle image and should be delivered via an index image
+LABEL com.redhat.delivery.operator.bundle: true
+
+# Define build arguments to receive dynamic values
+ARG FULL_COMMIT
+ARG GIT_URL
+
+LABEL name=aws-load-balancer-operator-bundle
+LABEL version="1.3.0"
+LABEL maintainer='NetworkEdge team <aos-network-edge-staff@redhat.com>'
+LABEL io.k8s.display-name='AWS Load Balancer Operator'
+LABEL io.k8s.description='Operator bundle for the AWS Load Balancer Operator'
+LABEL io.openshift.tags='networking,dns'
```

Comment:
This one looks like coming from EDO, can you check which tags were used in ALBO CPaaS?